Magento_CheckoutAgreements/web/js/mixins/checkout-agreements-mixin.js

```diff
@@ -0,0 +1,17 @@
+define([], function () {
+  'use strict';
+
+  return function (targetModule) {
+    return targetModule.extend({
+      htmlDecode: function (input) {
+        let e = document.createElement('div');
+        e.innerHTML = input;
+        return e.childNodes[0].nodeValue;
+      },
+
+      getCheckboxText: function (element) {
+        return this.htmlDecode(element.checkboxText);
+      }
```

Comment:
What do you think about renaming element for "data" or "viewModel" ? as we pass here actually "view model data object" not element.

Reply:
good point, I'll change it